macos/Sources/Features/Terminal/TerminalController.swift

```diff
@@ -574,6 +574,23 @@ class TerminalController: BaseTerminalController {
         window.close()
     }
 
+    @IBAction func returnToDefaultSize(_ sender: Any) {
+        guard
+            let window = NSApp.keyWindow,
+            let initialWidth = ghostty.config.windowWidth,
```

Comment:
These are in grid cells. So we can't simply read the config. This is unfortunately not the correct approach.